build.gradle.kts

```diff
@@ -248,6 +248,7 @@ project(":") {
     val testOutput = configurations.create("testOutput")
 
     dependencies {
+        compileOnly(kotlin("stdlib-jdk8"))
```

Comment:
Currently, you add this dependency only for core module. As a result, you can't navigate to sources of kotlin stdlib in, for example, `clion` module
Let's move it into `allprojects` section to add this dependency for all our modules (including future ones)